src/index.js

```diff
@@ -1,41 +1,40 @@
+const environment = require("./handlers/environment-handler");
+const AWS = require('aws-sdk')
+const lambda = new AWS.Lambda()
+const deployEnv = environment.environment;
+const querystring = require("querystring");
+
 exports.slack_commands_sync = async (event, context) => {
-    if (event === null || event === undefined || event.body === undefined) {
+    if (event["detail-type"] === "Scheduled Event") {
         console.log("Keep function hot");
         return;
     }
 
-    const body = require("querystring").parse(Buffer.from(event.body, "base64").toString());
+    const body = querystring.parse(event.body)
 
-    let url;
-    if (context.invokedFunctionArn.split(":")[7] !== "production") {
-        console.log("Sending to minerva slackCommandsAsync development...");
-        url = "https://g34h315ctk.execute-api.us-east-1.amazonaws.com/development/minerva-slackCommandsAsync-rzdj53m68JE9";
-    } else {
-        console.log("Sending to minerva slackCommandsAsync production...");
-        url = "https://bbfl9ivf5k.execute-api.us-east-1.amazonaws.com/production/minerva-slackCommandsAsync-rzdj53m68JE9";
+    console.log(`Sending to minerva-${deployEnv}-slackCommandsAsync...`);
+
+    const params = {
+        FunctionName: `minerva-${deployEnv}-slackCommandsAsync`,
+        InvocationType: "Event",
+        Payload: JSON.stringify(body),
     }
 
     await new Promise((resolve, reject) => {
-        const req = require("https").request(
-            url,
-            {
-                method: "POST",
-                headers: {
-                    "Content-Type": "application/json",
-                    "Content-Length": JSON.stringify(body).length,
-                },
-            },
-            res => {}
-        );
-        req.write(JSON.stringify(body));
-        req.end();
-        setTimeout(() => {
-            resolve();
-        }, 400);
-    });
+        lambda.invoke(params, (err, data) => {
+            if (err) {
+                console.log(err, err.stack)
+                reject()
+            } else {
+                setTimeout(() => {
+                    resolve();
+                }, 400);
+            }
+        })
+    })
 
     return {
-        statusCode: 200,
+        statusCode: 202,
```

Comment:
HTTP 202 (Accepted) would be more appropriate in this case, but I'm just being pedantic